client/my-sites/domains/email-providers-upsell/index.jsx

```diff
@@ -47,6 +47,7 @@ export default function EmailProvidersUpsell( { domain } ) {
 					comparisonContext="domain-upsell"
 					isDomainInCart={ true }
 					selectedDomainName={ domain }
+					selectedIntervalLength={ interval }
```

Comment:
@AllTerrainDeveloper: I'm surprised that we don't use `castIntervalLength()` here. Maybe that felt through the cracks?

Reply:
Great catch, @stephanethomas! I definitely missed this! 😳